iis-wf/iis-wf-affmatching/src/main/java/eu/dnetlib/iis/wf/affmatching/match/AffOrgMatcherFactory.java

```diff
@@ -97,12 +98,17 @@ public static AffOrgMatcher createDocOrgRelationMatcher(JavaSparkContext sparkCo
      * Creates {@link AffOrgMatchVoter}s for {@link #createDocOrgRelationMatcher(JavaSparkContext, String, String, String, Float)} 
      */
     public static ImmutableList<AffOrgMatchVoter> createDocOrgRelationMatcherVoters() {
+
+        FittingOrgWordsMatchVoter fitOrgWordsMatchVoter = new FittingOrgWordsMatchVoter(ImmutableList.of(','), 3, 0.8f, 0.9f);
```

Comment:
Well, we could of course generate it in mvn or something, and maybe one day we will do it. But for now it seems to me this is not very important thing. And this will only change when the test data changes and it won't very often.
But you're right, the best solution would be for it to work automatically.